### PR TITLE
Process method declaration (parameters)

### PR DESCRIPTION
**Pull request by ioanasv**

### CODE REVIEW

File: src/gr/uom/java/xmi/UMLModelPsiReader.java [hunk 1]
```diff
@@ -248,7 +251,18 @@ else if (isInterfaceMethod)
             umlOperation.setBody(null);
         }
 
-        //TODO: process return type, and parameters
+        List<KtParameter> parameters = methodDeclaration.getValueParameters();
+        for (KtParameter parameter : parameters) {
+            KtTypeReference typeReference = parameter.getTypeReference();
+            String paramName = getQualifiedName(parameter) != null ? getQualifiedName(parameter) : parameter.getText();
```
Comment (onewhl) [inline src/gr/uom/java/xmi/UMLModelPsiReader.java:257]: We should use just `parameter.getName();`

File: src/gr/uom/java/xmi/decomposition/VariableDeclaration.java [hunk 2]
```diff
@@ -94,11 +98,33 @@ public VariableDeclaration(CompilationUnit cu, String filePath, SingleVariableDe
...
+	public VariableDeclaration(KtFile ktFile, String filePath, KtParameter fragment) {
+		this.annotations = new ArrayList<UMLAnnotation>();
+		KtModifierList extendedModifiers = fragment.getModifierList();
+        if (extendedModifiers != null) {
+			List<KtAnnotation> annotations = extendedModifiers.getAnnotations();
+			annotations.forEach(ktAnnotation -> this.annotations.add(new UMLAnnotation(ktFile, filePath, ktAnnotation)));
+		}
+		//TODO check for the code element type
+		this.locationInfo = new LocationInfo(ktFile, filePath, fragment, CodeElementType.SINGLE_VARIABLE_DECLARATION);
+		this.variableName = String.valueOf(fragment.getNameIdentifier());
```
Comment (onewhl) [inline src/gr/uom/java/xmi/decomposition/VariableDeclaration.java:110]: We should use just `fragment.getName();`

File: src/gr/uom/java/xmi/decomposition/VariableDeclaration.java [hunk 3]
```diff
@@ -94,11 +98,33 @@ public VariableDeclaration(CompilationUnit cu, String filePath, SingleVariableDe
 		this.scope = new VariableScope(cu, filePath, startOffset, endOffset);
 	}
 
+	public VariableDeclaration(KtFile ktFile, String filePath, KtParameter fragment) {
```
Comment (onewhl) [inline src/gr/uom/java/xmi/decomposition/VariableDeclaration.java:101]: It should also work with KtField, so let's use KtElement instead of KtParameter

Comment (ioanasv) [reply src/gr/uom/java/xmi/decomposition/VariableDeclaration.java]: Done!

**Pull request merged**